Update elabapy for improved eLabFTW API (get_all_items)

JAC28 opened the pull request:
### Problem ### 
API was updated but package was not (see issue #17 
### Changes ###
added additional arguments in get_all_items method and changed params_name to 'params' instead of default 'data

JAC28 committed:
Changed method get_all_items to accept limit and offset as additional parameters

NicolasCARPi reviewed (changes requested):
Thank you for this. Shouldn't this be also the same for `get_all_experiments()`? Also, this change will break existing code, so params should have a default value to be backward compatible.

As for the param_name change, it needs to be `data` or `files` for the requests library. This should be made clear in a comment.

JAC28 committed:
Added API functionality to get_all_experiments and provide backwards functionality by default value limit=15 and offset=0

JAC28 committed:
Update Manager.py

Forgot to change the get.request

JAC28 commented:
Hi @NicolasCARPi, you've mententioned two valid points thus I've added the functionality for experiments as well and set default values. 
With param_name I am confused since it only works for me with `params`. This is also in accordance with the [doku](https://docs.python-requests.org/en/latest/api/#requests.get).

NicolasCARPi commented:
Yes you're correct for the `params` stuff (I didn't run your code yet). In fact it took me a few minutes to remember why I put this here... You can add a comment linking to the request doc in the function comment so it is clear for future devs, including us ;)

JAC28 commented on elabapy/Manager.py:
```diff
@@ -24,23 +24,23 @@ def create_template(self):
             This function returns a list of all experiments.
         """
-        return self.send_req("experiments/")
+        return self.send_req("experiments/", params, param_name='params')
```
param_name has to be 'params' to be in accordance to the requests [doku](https://2.python-requests.org/en/master/api/#requests.get)

JAC28 committed:
Update Manager.py

Added a brief comment for the content of params for get_all_items/experiments. Link to dokumentation of requests given for explanation of 'param_name'='params'

NicolasCARPi commented on elabapy/Manager.py:
```diff
@@ -27,6 +27,8 @@ def create_template(self):
         """
             This function returns a list of all experiments.
+            params is a dict with keys 'limit' and/or 'offset'
+            param_name has to be 'params' to be in accordance with the current requests-doku (https://2.python-requests.org/en/master/api/#requests.get)
```
use: "current requests documentation"

JAC28 replied:
I've added your suggestion within the last commit.

JAC28 committed:
Update Manager.py

Fixed typo

NicolasCARPi reviewed (approved):
Thanks for your contribution!
